### PR TITLE
JVM config overrides

### PR DESCRIPTION
**Pull request by razvan**
# Description
 
CI: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/hdfs-operator-it-custom/96/

Use JVM defaults explicitly.

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [x] Changes are OpenShift compatible
- [x] CRD changes approved
- [x] Helm chart can be installed and deployed operator works
- [x] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [x] Proper release label has been added
```

### CODE REVIEW

File: docs/modules/hdfs/pages/usage-guide/configuration-environment-overrides.adoc [hunk 1]
```diff
@@ -7,7 +7,14 @@ IMPORTANT: Overriding certain properties can lead to faulty clusters. In general
 
 == Configuration Properties
 
-For a role or role group, at the same level of `config`, you can specify `configOverrides` for the `hdfs-site.xml` and `core-site.xml`. For example, if you want to set additional properties on the namenode servers, adapt the `nameNodes` section of the cluster resource like so:
+For a role or role group, at the same level of `config`, you can specify `configOverrides` for the following files:
+
+- `hdfs-site.xml`
+- `core-site.xml`
+- `security.properties`
```
Comment (sbernauer) [inline docs/modules/hdfs/pages/usage-guide/configuration-environment-overrides.adoc]:
```suggestion
- `hdfs-site.xml`
- `core-site.xml`
- `hadoop-policy.xml`
- `ssl-server.xml`
- `ssl-client.xml`
- `security.properties`
```

**Review by sbernauer — CHANGES_REQUESTED** [review]

**Comment by sbernauer** [comment]
Note that the defaults with 30s are pretty high (e.g. for journalnodes and namenodes) but if it works that's great!

**Comment by razvan** [comment]
These are the JVM defaults that some customers use already. They are open for discussion of course.

**Review by sbernauer — APPROVED** [review]
Yeah, good point. Lets see how it goes!

**Pull request merged**